chef-provisioning-azure.gemspec

```diff
@@ -15,10 +15,10 @@ Gem::Specification.new do |s|
   s.homepage = 'https://github.com/chef/chef-provisioning-azure'
   s.license = 'Apache-2.0'
 
-  s.add_dependency 'chef', '>= 12.0'
   s.add_dependency 'chef-provisioning', '~> 1.0'
   s.add_dependency 'stuartpreston-azure-sdk-for-ruby', '~> 0.7'
 
+  s.add_development_dependency 'chef', '>= 12.0'
```

Comment:
bundler doesn't pull development dependencies transitively.  A development dep is what you would need if you were actively developing that gem - so the dev deps of chef-provisioning don't get installed when you bundle install this gem.  Since I'm yanking chef as a dep of both chef-provisioning and for cheffish (which is a real dependency of chef-provisioning), chef never gets pulled in unless explicitly added as a development dependency.